### ISSUE TITLE
[Bug]: Contextual menu , two numbers are displaying for split submenus on voice command "Show numbers".

### ISSUE THREAD

**Issue by gilenzi** [microsoft/fluentui#31455]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
System:
    OS: Windows 10 10.0.22631
    CPU: (12) x64 13th Gen Intel(R) Core(TM) i5-1335U
    Memory: 4.67 GB / 15.69 GB
  Browsers:
    Edge: Chromium (124.0.2478.51)
    Internet Explorer: 11.0.22621.3527
```

### Are you reporting Accessibility issue?

Yes

### Reproduction

https://scc-storybook.azurewebsites.net/?path=/story/templates-advancedfilterlistpage--default&args=filterConfig:AdvancedFilterWithSaveQueries

## Actual Behavior
By using "Show numbers" command two numbers are displaying for Contextual menu  component on it's split submenus. Activating any of the number same action is getting performed.
## Expected Behavior
By using voice access command only one number should be displayed for submenu.

![image](https://github.com/microsoft/fluentui/assets/78443187/0e0592f4-38d0-4f96-8b65-31c55f4eeff9)

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by smhigley** [comment]
@gilenzi this appears to be using the v8 ContextualMenu, not v9 controls:
<img width="1058" alt="screenshot of UI referenced in the original comment, with the elements panel open highlighted an element with a ContextualMenu className" src="https://github.com/microsoft/fluentui/assets/3819570/40c8df65-16dc-41e8-a751-ab043b86c3c7">

Let me know if I'm misunderstanding that, or inspecting the wrong thing.

Assuming this is v8, this is a known issue that stems from the v8 behavior of having a single focus stop for two interactive controls (the primary action and menu button). To make this meet minimum bar accessibility across keyboard, desktop screen readers, touch screen readers, and voice control, we had to create nested buttons for the individual actions. This issue should be marked as by design, and it will go away when updating to v9.

There is more information on our [internal known issues wiki](https://dev.azure.com/microsoftdesign/fluent-ui/_wiki/wikis/fluent-ui.wiki/332/Fluent-v8-known-issues?anchor=splitbuttons-and-split-menuitems-have-nested-interactives%2C-or-double-voice-control-numbers).
